common/web/gesture-recognizer/src/engine/touchEventEngine.ts

```diff
@@ -76,8 +76,9 @@ export class TouchEventEngine<ItemType, StateToken = any> extends InputEventEngi
     super.dropTouchpoint(source);
 
     for(const key of Object.keys(this.safeBoundMaskMap)) {
-      if(this.getTouchpointWithId(Number.parseInt(key, 10)) == source) {
-        delete this.safeBoundMaskMap[key];
+      const keyVal = Number.parseInt(key, 10)
+      if(this.getTouchpointWithId(keyVal) == source) {
+        delete this.safeBoundMaskMap[keyVal];
```

Comment:
Is this a bug fix?

Reply:
Legal type:  `number'
Original type: `string`.

JS is perfectly happy to do type-coercion on our behalf here, but with the enhanced settings, TypeScript doesn't care and will require us to do the type-coercion ourselves.  There was no bug here.